lib/github.ts

```diff
@@ -24,32 +24,9 @@ type Options = {
   useFallback?: boolean
 }
 
-export const ReleaseBranches = ['master', 'main']
-export const DevelopBranches = ['develop']
-
-export async function getShaFromRef({
-  octokit,
-  owner,
-  repo,
-  ref,
-  useFallback = true,
-}: Options) {
-  if (useFallback && ref === 'develop') {
-    const availableBranches = (
-      await octokit.rest.repos.listBranches({
-        owner,
-        repo,
-      })
-    ).data.map((branch) => branch.name)
-    if (!availableBranches.includes('develop')) {
-      const fallback = availableBranches.includes('main') ? 'main' : 'master'
-      warning(
-        `Branch "develop" not available in repository "${owner}/${repo}", falling back to "${fallback}".`
-      )
-      ref = fallback
-    }
-  }
+export const STANDARD_BRANCH = 'main'
```

Comment:
@hongaar - I removed this bit from getShaFromRef as all components now just use a main branch. Is this ok?

Reply:
see function below this comment